anymaniax/orval
# Fix Query parameters reactivity in Vue

## Status

**READY/WIP**

Fix #930 

## Description

Among other minor fixes this addresses #930 by unwrapping `params` mayberef

## Todos


## Steps to Test or Reproduce

In the `vue-query` sample, open dev tools, refresh the page, see first request with filter=2 and then a few seconds later request for filter=3. This confirms that reactivity works. If we don't unwrap params - we'll get this error:

```
query.ts:444 TypeError: Converting circular structure to JSON
    --> starting at object with constructor 'ReactiveEffect'
    |     property 'computed' -> object with constructor 'ComputedRefImpl'
    --- property 'effect' closes the circle
    at JSON.stringify (<anonymous>)
    at parseValue (buildURL.js:51:20)
    at Object.forEach (utils.js:245:10)
    at serialize (buildURL.js:47:13)
    at Object.forEach (utils.js:251:12)
    at buildURL (buildURL.js:36:11)
    at dispatchXhrRequest (xhr.js:44:47)
    at new Promise (<anonymous>)
    at xhrAdapter (xhr.js:15:10)
    at dispatchRequest (dispatchRequest.js:58:10)
```
